src/utilities/getCollateralValue.ts

```diff
@@ -0,0 +1,12 @@
+import BigNumber from 'bignumber.js';
+import { Asset } from 'types';
+import { convertWeiToCoins } from './common';
+
+const getCollateralValue = ({ asset, amountWei }: { asset: Asset; amountWei: BigNumber }) => {
```

Comment:
This is a minor thing, but to follow the same nomenclature as the other utils perhaps this one should be named `calculateCollateralValue`?